src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibratorArgumentCollection.java

```diff
@@ -0,0 +1,88 @@
+package org.broadinstitute.hellbender.tools.walkers.vqsr;
+
+
+import org.broadinstitute.hellbender.cmdline.Argument;
+
+/*
+ * A bunch of annotations for VQSR.
+ */
+final class VariantRecalibratorArgumentCollection {
+
+    static public enum Mode {
+        SNP,
+        INDEL,
+        BOTH
+    }
```

Comment:
can't find any that would be suitable here. SAMPileUpFeature seems close but it has `VariantType { NONE, SNP, INSERTION, DELETION, INDEL }`